scripts/release.js

```diff
@@ -1,211 +1,168 @@
-  logger.log('Reseting HEAD to origin/main');
-  spawnOrFail('git', ['reset --hard origin/main']);
-}
-spawnOrFail('git', [' clean -ffxd .']);
-
-
-logger.log(`Updating ${versionFile} with ${versionString}`);
-
-// Update version number in Versions file.
-fs.writeFileSync(
-  versionFile,
-  fs
-    .readFileSync(versionFile)
-    .toString()
-    .replace(/return '[.0-9]+';/, `return '${versionString}';`)
-);
-
-logger.log(`Updating CHANGELOG.md with a new release entry - ${versionString}`);
-// Update CHANGELOG.md with release version
-fs.writeFileSync(
-  changeLog,
-  fs
-    .readFileSync(changeLog)
-    .toString()
-    .replace(
-      /\[Unreleased\]/,
-      `[${versionString}] - ${new Date().toISOString().slice(0, 10)}`
-    )
-);
-spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
-logger.log(`Updated package.json version to ${versionString}`);
-
-const updatedSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`]).trim();
-
-// Skip updating peer dependencies for hotfix
-if (release_option !== '5') {
+
+const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutput: true })).trim();
+
+const deployDemo = (version) => {
+  logger.log('Deploying unique release candidate Meeting Demo URL...');
+  // Checkout the meeting demo
+  process.chdir(path.join(__dirname, '../..'));
+  if (!fs.existsSync('amazon-chime-sdk')) {
+    spawnOrFail('git', ['clone --branch meeting-dev https://github.com/aws-samples/amazon-chime-sdk.git']);
+  } else {
+    spawnOrFail('rm', ['-rf amazon-chime-sdk']);
+    spawnOrFail('git', ['clone --branch meeting-dev https://github.com/aws-samples/amazon-chime-sdk.git']);
+  }
+
+  // Update the demo to the most up to date version of JS SDK
+  process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
+  updateDependency(`amazon-chime-sdk-js@${latestNPMJSSdkVersion}`);
+  process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
+  updateDependency('amazon-chime-sdk-component-library-react', `../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-${version}.tgz`);
+
+  process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting/serverless'));
+  const formattedVersion = version.replace(/\./g, '-');
+  spawnOrFail('node', [`./deploy.js -r us-east-1 -b chime-sdk-components-demo-${formattedVersion} -s chime-sdk-components-demo-${formattedVersion}`], { printErr: true });
+
+  process.chdir(path.join(__dirname, '..'));
+  spawnOrFail('rm', ['-rf ../amazon-chime-sdk']);
+};
+
+const getCurrentRemoteBranch = () => {
+  return (spawnOrFail('git', ['for-each-ref --format="%(upstream:short)" "$(git symbolic-ref -q HEAD)"'], { skipOutput: true })).trim();
+};
+
+const buildAndPack = () => {
+  logger.log('Building package...');
+  spawnOrFail('npm', [`run build:publish`]);
+  logger.log('Packaging ...');
+  spawnOrFail('npm', ['pack'], { printErr: true });
+};
+
+const updateJSSdkDep = () => {
```

Comment:
Request to check comment: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/714/files#r826419554.


Should we also take `isPreRelease` version into consideration while installing JS SDK?